src/test/java/SmokeTest.java

```diff
@@ -0,0 +1,87 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.*;
+
+import java.io.IOException;
+
+public class SmokeTest extends BaseTest{
+
+    private String emailAddress = randomEmail();
+
+    private String password = randomPassword();
+    private String actualProductName;
+    private double actualPrice;
+    private int orderId;
+    private String productName = "blue jacket";
+    private String thankYouText = "Thank you for your purchase!";
+    private String accountIsCreated = "Thank you for registering with Fake Online Clothing Store.";
+
+    @BeforeTest
+    public void setUp() throws IOException {
+        driver = initializeDriver();
+    }
+
+    @Test
+    public void smokeTest() throws InterruptedException, IOException {
+
+        //open homepage and enter searched word
+        HomePage homePage = new HomePage(driver);
+        homePage.goToHomepage();
+        homePage.enterSearchedWord(productName);
+
+        SearchPage searchPage = new SearchPage(driver);
+        Assert.assertTrue(searchPage.verifySearchedText(productName));
+        searchPage.clickOnProductImg();
+
+        //add to cart the searched product and verify correct quantity
+        ProductPage productPage = new ProductPage(driver);
+        actualProductName = productPage.getActualProductName();
+        actualPrice = productPage.getActualPrice();
+
+        productPage.addToCart("M", "Blue");
+        Assert.assertTrue(productPage.verifyAddButtonIsChangedToAdded());
+        productPage.openCartPopup();
+        Assert.assertTrue(productPage.verifyIsQtyCorrect("1"));
+
+        productPage.clickOnProceedToCheckout();
+
+        //fill the shipping form with valid data
+        ShippingPage shippingPage = new ShippingPage(driver);
+        shippingPage.enterEmail(emailAddress);
+        Assert.assertTrue(shippingPage.isLoading());
+
+//        shippingPage.fillShippingAddressForm(getDetailsMap("src/test/java/data/shippingFormDetails.json"));
+        shippingPage.fillShippingAddressForm(DataReader.readShippingForm("src/test/java/data/shippingFormDetails.json"));
+
+        //choose shipping method
+        shippingPage.chooseShippingMethods("tablerate_bestway");
```

Comment:
Again these 2 strings are fairly large and should not be hardcoded in. Place them in a utilities class.